### DISCUSSION TITLE
workflow_run trigger details not in Actions Summary UI

### DISCUSSION THREAD

**Discussion by kriswuollett** [community/community#12610] [Actions]
I've set up a workflow to be triggered via `workflow_run`. In the UI it shows "Triggered via workflow run ${TIME} ago, ${USER} completed - ${GIT_SHA}":

![CleanShot 2022-03-09 at 12 55 29@2x](https://user-images.githubusercontent.com/1834192/157502050-5288d229-f831-4435-8e44-f3c2f4e1c60e.png)

Shouldn't the template for that part of the UI show "Triggered via workflow run ${TIME} ago, ${WORKFLOW_TITLE}/${WORKFLOW_RUN_URL} - ${GIT_SHA}." 

There isn't a way to directly click to the triggering run to see the uploaded artifacts which is helpful to debug the downstream workflow that got triggered. Also who triggered the upstream workflow isn't directly relevant, that should just show up in the UI for the workflow that started the whole `workflow_run` chain.

**Comment by ethomson** [comment]
Thanks for the feedback @kriswuollett - I'll talk with the team about how we might be able to improve visibility and link to the other workflow.
